### DISCUSSION TITLE
Slider value

### DISCUSSION THREAD

**Discussion by RealCoChenchao** [Appsilon/shiny.fluent#133] [General]
Love this framework and package, thank you!

A quick question about get the values from slider. Currently, I believe it runs every step from starting value to the ending value. In another word, there is a slider from 0 to 100 with step 1. If I move it from 2 to 6, value 3, 4, 5 and 6 will be picked. How can I only get the final value, which is 6 here? Is `onChanged` for this functionality?

**Comment by jakubsob** [comment]
Hi @RealCoChenchao

we're still working on implementing this behavior - it should be available in the next version of the package. At this moment the solution would be to use [`shiny::debounce`](https://shiny.rstudio.com/reference/shiny/1.0.4/debounce.html) to reduce the number of changes of input value. Please see the following example how you can use it with the slider.

```r
library(shiny)
library(shiny.fluent)
library(magrittr)

ui <- fluentPage(
  Slider.shinyInput(inputId = "slider", value = 5, min = 0, max = 10),
  textOutput("value")
)

server <- function(input, output, session) {
  slider_debounced <- reactive(input$slider) %>% debounce(200)
  output$value <- renderText({
    slider_debounced()
  })
}

shinyApp(ui, server)
```
